packages/core/admin/admin/src/content-manager/components/CollectionTypeFormWrapper/index.js

```diff
@@ -237,6 +237,8 @@ const CollectionTypeFormWrapper = ({ allLayoutData, children, slug, id, origin }
         return Promise.reject(err);
       }
     },
+    // TODO: remove when we evaluate the need of the useCallback
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
this again, im not a fan of slapping eslint rules as an easy win it's really bad practice.

Reply:
ok I understand but as I said in the comment, I will remove it in this activity https://strapi-inc.atlassian.net/browse/DX-570
if you are not ok with it I need to use again the getFetchClient

Reply:
I don't understand why you don't think you can include it in the deps? It's a stable function because it lives outside of react no?

Reply:
Looking at the hook quickly if anything you need to wrap the object you return in a useMemo? But we can't have this many eslint disables around hook deps, we'll be shouting ourselves in the foot.

Reply:
I am fine with your comment, I can try to solve my problem in a different way. Just to keep in mind, there are other places, into the code, where we are using the eslint comment

Reply:
> Just to keep in mind, there are other places, into the code, where we are using the eslint comment

Then we should as a collective try get rid of them 🤝

Reply:
could be an interesting discussion for the Frontend sync @simotae14 @joshuaellis 
we should work on enforcing some general rules from now and forward